src/FormInput.php

```diff
@@ -5,7 +5,7 @@
 use lewiscowles\Rfc\AbstractNode;
 
 
-Final class FormInput extends AbstractNode {
+final class FormInput extends AbstractNode {
```

Comment:
specificity over caps... This really should come with a coding style issue and PR first

Reply:
It is the same as the previous comment.